added new topic: Sharing team configuration files

JamesBauman opened the pull request:
@t1m0thyj 

Request review so that we can push the new topic into the preview branch.

t1m0thyj commented on docs/user-guide/cli-using-sharing-team-config-files.md:
````diff
@@ -0,0 +1,98 @@
+2. Issue the following command:
+
+    ```
+    zowe config import <DriveLetter>:\<SharedZoweConfig>\zowe.config.json
````
"SharedZoweConfig" was an example of a path. For the general name maybe we want to say `<FolderPath>`?

t1m0thyj commented on docs/user-guide/cli-using-sharing-team-config-files.md:
```diff
@@ -0,0 +1,98 @@
+
+        The directory path on the drive
+
+    **Note:** You can specify any path that Windows Explorer can access. For example, a UNC network path (`\\<HostName>\SharedZoweConfig\zowe.config.json`) or local file path (`C:\Users\<UserName>\Downloads\zowe.config.json`).
```
Sorry I think I mentioned Windows Explorer here, but we should probably use a more generic cross-platform term. Maybe "your file manager" instead of "Windows Explorer". Or "your file manager (e.g., Windows Explorer)" if we want to keep mentioning it.

JamesBauman commented:
@t1m0thyj 

Updated per comments,
Added code to Access to LPARs that access services through one API Mediation Layer using certificate authentication

You can view the articles here:

https://github.com/zowe/docs-site/blob/jabauman-cli-ZE-v2/docs/user-guide/cli-using-sharing-team-config-files.md

https://github.com/zowe/docs-site/blob/jabauman-cli-ZE-v2/docs/user-guide/cli-using-team-configuration-team-leaders.md#access-to-lpars-that-access-services-through-one-api-mediation-layer

thx

t1m0thyj commented on docs/user-guide/cli-using-team-configuration-team-leaders.md:
```diff
@@ -204,10 +205,49 @@ The following example illustrates that the settings access multiple services usi
+                "certFile": "./zowe-cert.pem",
+                "certKeyFile": "./zowe-cert-key.pem",
+                "host": "example.com",
+                "port": 7554,
```
Can the port in the above example (line 189) also be changed to 7554 to match? The default port for z/OSMF is 443, but for API ML is 7554.

JamesBauman commented:
@t1m0thyj 

Update the port # in line 189.
